packages/page-staking/src/Query/Validator.tsx

```diff
@@ -28,11 +29,11 @@ function Validator ({ className = '', validatorId }: Props): React.ReactElement<
   );
 }
 
-export default React.memo(styled(Validator)`
+export default React.memo(styled(Validator)(({ theme }: ThemeProps) =>`
   .staking--Chart {
-    background: white;
+    background: ${theme.theme};
```

Comment:
Just wondering if this should not rather be the table background? (Still has the same effect, but aligns closer wit the initial where the background is different from the page background)

Reply:
Ok. I'll update and see the result.

Thanks.

Reply:
@jacogr I updated the PR to use the table variables instead, and this is the new look.

<img width="1440" alt="Screenshot 2021-05-30 at 10 51 07" src="https://user-images.githubusercontent.com/3910877/120098579-feabd180-c136-11eb-9158-f0fa48e5dbf5.png">